test/changelog-notes/default-changelog-notes.ts

```diff
@@ -20,12 +20,16 @@ import {
   safeSnapshot,
 } from '../helpers';
 import {DefaultChangelogNotes} from '../../src/changelog-notes/default';
-import {parseConventionalCommits} from '../../src/commit';
+import {ConventionalCommit, parseConventionalCommits} from '../../src/commit';
 import {PullRequestBody} from '../../src/util/pull-request-body';
 import {Version} from '../../src/version';
+const presetFactory = require('conventional-changelog-conventionalcommits');
+
+const COMMIT_WITH_AUTHOR_FORMAT = '{{#if authors}} by {{authors}}\n {{~/if}}\n';
```

Comment:
How to add PR author to a commit entry for the generated release notes.

I wonder if this feature be default or opt-in.